conan.cmake

```diff
@@ -318,15 +320,15 @@ function(conan_cmake_install)
         else()
             set(CONAN_BUILD_POLICY ${CONAN_BUILD_POLICY} --build=${ARG})
         endif()
-    endforeach()
+    endforeach()   
     if(ARGUMENTS_CONAN_COMMAND)
        set(conan_command ${ARGUMENTS_CONAN_COMMAND})
     else()
-      set(conan_command conan)
+      set(conan_command "C:\\bincrafters\\conan\\conan.bat")
```

Comment:
This doesn't look right to me.

Reply:
yeah, my fault, I didn't want to commit this, removed